generate-release/src/release_notes.rs

```diff
@@ -107,3 +135,72 @@ file_name = "{file_name}.md"
         title = title.trim().replace('"', "\\\"")
     )
 }
+
+#[allow(clippy::too_many_arguments)]
+fn generate_and_open_issue(
+    client: &GithubClient,
+    issue_titles: &HashSet<String>,
+    pr: &GithubIssuesResponse,
+    title: &str,
+    authors: &[String],
+    file_path: &Path,
+    dry_run: bool,
+) {
+    let pr_number = pr.number;
+    let issue_title = format!("Write release notes for PR #{pr_number}: {title}");
+
+    // Check if this issue already exists
+    // If it does, we don't want to spam the repo with duplicate issues
+    if issue_titles.contains(&issue_title) {
+        println!("Issue already exists for PR #{}: {}", pr_number, title);
+        return;
+    }
+
+    let pr_url = format!("https://github.com/bevyengine/bevy/pull/{pr_number}",);
+    let file_path = file_path.to_string_lossy();
+
+    let authors = authors.join(", ");
+
+    // The weird indentation is intentional.
+    // Otherwise, the text is not formatted correctly on GitHub as the tabs are copied into the issue body.
+    let issue_body = format!(
+        "{pr_url} needs release notes for the upcoming Bevy release!
+
+Original authors: {authors}
+
+Please reply below if you'd like to write these notes. 
+While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature,
+anyone can contribute release notes!
```

Comment:
I'm pretty sure GitHub-flavored Markdown doesn't ignore newlines, so this is actually spread across two separate lines.
```suggestion
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!
```

Reply:
Yeah it does separate the lines: 
<img width="737" alt="Screenshot of the text being rendered across two lines instead of one" src="https://github.com/bevyengine/bevy-website/assets/49806985/f01a2960-561b-4b04-bdf0-beb20fd7850d">